std/typecons.d

```diff
@@ -518,6 +472,23 @@ template Tuple(Specs...)
             // This is mostly to make t[n] work.
             alias _Tuple_super this;
         }
+        ///
+        unittest
+        {
+            auto t = tuple(1, " hello ", 2.3);
+            writeln(t);        // Tuple!(int, string, double)(1, " hello ", 2.3)
+            writeln(t.expand); // 1 hello 2.3
```

Comment:
The `writeln`s probably need `import std.stdio;`. However, being a unittest, it's probably better to use asserts instead. For example:
````
import std.format : format;
assert(format("%s", t) == "Tuple!(int, string, double)(1, " hello ", 2.3)");
````
and so on.